packages/features/ee/workflows/components/WorkflowDetailsPage.tsx

```diff
@@ -1,19 +1,18 @@
-import { ArrowDownIcon } from "@heroicons/react/outline";
 import { WorkflowActions, WorkflowTemplates } from "@prisma/client";
```

Comment:
All diffs in this page are because the v1 version had the same name, and I moved v2 to the same place. So this is actually the difference between v1 and v2.